identity-apps-core/apps/authentication-portal/src/main/webapp/includes/layout-resolver.jsp

```diff
@@ -80,27 +80,15 @@
                         String temp = brandingPreference.getJSONObject(LAYOUT_KEY).getString(ACTIVE_LAYOUT_KEY);
                         if (StringUtils.equals(temp, PREFIX_FOR_CUSTOM_LAYOUT_NAME)) {
-                            String tempLayout = "";
-                            String tempLayoutFileRelativePath = "";
-                            String tempBaseURL = "";
-
                             // App-wise and tenant-wise custom layout resolving logic.
                             if (StringUtils.equals(preferenceResourceType, APP_PREFERENCE_RESOURCE_TYPE)) {
-                                tempLayout = temp + CUSTOM_LAYOUT_NAME_SEPERATOR + tenantRequestingPreferences + CUSTOM_LAYOUT_NAME_SEPERATOR + convertApplicationName(applicationRequestingPreferences);
-                                tempLayoutFileRelativePath = layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences) + "/apps/" + convertApplicationName(applicationRequestingPreferences) + "/body.ser";
-                                tempBaseURL = layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences) + "/apps/" + convertApplicationName(applicationRequestingPreferences);
-                            }
-
-                            if (config.getServletContext().getResource(tempLayoutFileRelativePath) == null || StringUtils.equals(preferenceResourceType, ORG_PREFERENCE_RESOURCE_TYPE)) {
-                                tempLayout = temp + CUSTOM_LAYOUT_NAME_SEPERATOR + tenantRequestingPreferences;
-                                tempLayoutFileRelativePath = layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences) + "/body.ser";
-                                tempBaseURL = layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences);
-                            }
-
-                            if (config.getServletContext().getResource(tempLayoutFileRelativePath) != null) {
-                                layout = tempLayout;
-                                layoutFileRelativePath = tempLayoutFileRelativePath;
-                                layoutData.put("BASE_URL", tempBaseURL);
+                                layout = temp + CUSTOM_LAYOUT_NAME_SEPERATOR + tenantRequestingPreferences + CUSTOM_LAYOUT_NAME_SEPERATOR + convertApplicationName(applicationRequestingPreferences);
+                                layoutFileRelativePath = layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences) + "/apps/" + convertApplicationName(applicationRequestingPreferences) + "/body.ser";
+                                layoutData.put("BASE_URL", layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences) + "/apps/" + convertApplicationName(applicationRequestingPreferences));
+                            } else if (config.getServletContext().getResource(tempLayoutFileRelativePath) == null || StringUtils.equals(preferenceResourceType, ORG_PREFERENCE_RESOURCE_TYPE)) {
```

Comment:
```suggestion
                            } else if (StringUtils.equals(preferenceResourceType, ORG_PREFERENCE_RESOURCE_TYPE)) {
```
Let's remove the `config.getServletContext().getResource(tempLayoutFileRelativePath) == null` part. Since we removed `tempLayoutFileRelativePath`, this can generate a runtime error.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5522/commits/48127e9327030393f0e99f486cc09640bc6d80b1